presto-main/src/test/java/com/facebook/presto/sql/planner/TestLogicalPlanner.java

```diff
@@ -315,7 +315,7 @@ public void testCorrelatedScalarAggregationRewriteToLeftOuterJoin()
                                                 any(
                                                         aggregation(ImmutableMap.of("PARTIAL_COUNT", functionCall("count", ImmutableList.of("NON_NULL"))),
                                                                 any(
-                                                                        join(LEFT, ImmutableList.of(), Optional.of("ORDERKEY = 3"),
+                                                                        join(LEFT, ImmutableList.of(), Optional.of("3 = ORDERKEY"),
```

Comment:
This comes from SimplifyExpressions, which is now called as part of predicate push down for joins without equality predicates.  I believe it is deterministic.